llama_stack/cli/model/prompt_format.py

```diff
@@ -38,8 +38,8 @@ def _add_arguments(self):
             "-m",
             "--model-name",
             type=str,
-            default="llama3_1",
```

Comment:
Is this removal intentional?

Reply:
Ah just saw this in PR description:
> Removes the default value and field for --model-name to prevent users from mistakenly thinking a model family name is acceptable

Reply:
Yes, the default references a model family, not a model name. Is there a model name you'd rather have as default instead? Otherwise, I thought the default field should be remvoed

Reply:
should we elevate model-family as a top-level concept in llama-stack?

Reply:
I like that idea

Reply:
@raghotham I wanted to follow-up on this. This PR just focuses on fixing the prompt format command to be tied to the model name. Can we get this one in as is or is there anything else you would like to see out of this PR?